src/components/TradeWidget/FormMessage.tsx

```diff
@@ -39,6 +46,45 @@ const FormMessage = styled.div.attrs<{ className?: string }>(props => ({
     box-sizing: border-box;
     margin: 0.3rem 0 1rem;
   }
+  &.hidden {
+    visibility: hidden;
```

Comment:
Why switching between using classes for css and props.
Stick to one. Like `visibility: ${props => props.visible ? 'hidden' : 'visible'}`

Or don't. Depends on semantics I guess

What's your opinion?

Reply:
i was having issues with the `Styled` syntax of `styled.div.attrs<{ ... }>` and adding my own props - it was throwing in TS. Any advice? An example would be useful

Reply:
[example](https://github.com/gnosis/dex-react/blob/67f578ce5f0187641297b158e86cca1e2010779d/src/components/TradeWidget/OrderValidity.tsx#L87)

Reply:
No I mean when it's a styled component using `.attrs`

Reply:
will not change this for now, thanks tho